temp.c

```diff
@@ -2,13 +2,16 @@
 #include <stdio.h>
 #include <string.h>
 #include <stdlib.h>
+#include <ctype.h>
+
+#define konverzija_f ((1.8 * celsius) + 32)
+#define konverzija_c ((5.0/9.0) * (fahrenheit-32))
 
```

Comment:
Ovo sto ste ovde definisali, zove se macro. Mozete izguglati malo o njima, a imate i u onim knjigama sto sam vam poslao. Nisam to hteo. Hteo sam da napravite dve funkcije. Ja cu ovde dati primer za jednu.

float konvert_iz_cels_u_farh(float cels) {
   return (1.8 * celsius) + 32;
}

Potrebno je i da ove konstante definisete preko define.